bld2repo/bld2repo/__init__.py

```diff
@@ -45,7 +45,6 @@ def get_buildrequire_pkgs_from_build(build_id, session, config):
         for rpm in tagged_rpms:
             if pkg["build_id"] == rpm["build_id"] and rpm["arch"] in archs:
                 pkg_md["rpms"].append(rpm)
-                tagged_rpms.remove(rpm)
```

Comment:
Not sure if it is a good idea to remove items from the currently iterated list.

Reply:
Indeed, it's not a good idea. That's why I removed this line which removes items from the currently iterated list. :-)
During testing, the missing packages/RPMs reported by @kdudka appeared to be downloaded. So it seems to fix the issue.
![obrazek](https://user-images.githubusercontent.com/23337584/151758716-099c0b33-8098-40d1-b849-040eda533714.png)